src/components/Avatar/SwAvatar.vue

```diff
@@ -16,8 +16,7 @@ const props = defineProps({
   square: { type: Boolean, default: false },
   border: { type: Boolean, default: false },
 });
-
-const letter = computed(() => props.text.charAt(0).toUpperCase());
+const letter = computed(() => (props.text || ' ').charAt(0).toUpperCase());
```

Comment:
Why not giving `' '` as the prop's default value?

Reply:
because I think people are passing an empty string as the text prop (by accident)